charts/generic-service/values.yaml

```diff
@@ -219,3 +219,9 @@ scheduledDowntime:
   # Stop at 10pm UTC Monday-Friday
   shutdown: '0 22 * * 1-5'
   serviceAccountName: scheduled-downtime-serviceaccount
+  # every 10 minutes between 7am and 10pm UTC Monday-Friday
```

Comment:
this means there will be no retries for an hour after restart - not sure how much of a big deal this is but thought it better to do this than risk a failing job by starting it at 6.